src/examples/recognised-overseas-pension-schemes-notification-list/example/index.njk

```diff
@@ -0,0 +1,87 @@
+---
+  layout: layout-example.njk
+---
+
+<h3 class="govuk-heading-l">Austria</h3>
+
+<div class="call-to-action">
+<p class="govuk-body">The requirements to be a ROPS changed from 6 April 2017. You’ll need to check that the scheme you’re transferring to on or after that date meets the new requirements.<br><br></p>
+
+<p class="govuk-body">HMRC cannot guarantee these are ROPS or that any transfers to them will be free of UK tax. It’s your responsibility to find out if you have to pay tax on any transfer of pension savings. Find out more about <a href="https://www.gov.uk/government/collections/overseas-pension-schemes" class="govuk-link">Overseas pension schemes</a>.</p>
+</div>
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{{
+  govukTable({
+    head: [
+      {
+        text: 'ROPS'
+      },
+      {
+        text: 'Country'
+      }
+    ],
+    rows: [
+      [
+        {
+          text: 'APK Pensionskasse AG'
+        },
+        {
+          text: 'Austria'
+        }
+      ],
+      [
+        {
+          text: 'Bonus Pensionskassen Aktiengesellschaft'
+        },
+        {
+          text: 'Austria'
+        }
+      ],
+      [
+        {
+          text: 'Sozialversicherungspensionskasse AG'
+        },
+        {
+          text: 'Austria'
+        }
+      ],
+      [
+        {
+          text: 'VBV-Pensionskasse Aktiengesellschaft'
+        },
+        {
+          text: 'Austria'
+        }
+      ]
+
+    ]
+  })
+}}
+
+<br /><h3 class="govuk-heading-l">Changes for Austria at 1 July 2018</h3>
+
+{{
```

Comment:
For this table (and possibly the others as well) we should be using a table caption as the example Nunjucks code shows at https://design-system.service.gov.uk/components/table/

Reply:
Table caption doesn't seem available in markdown https://github.com/hmrc/design-system/pull/223

Reply:
Yeah, just had a look and it seems like table captions in Markdown down are pretty patchy across all the different types of Markdown. A heading is good second option.